data/mods/Magiclysm/Spells/animist.json

```diff
@@ -89,9 +89,9 @@
     "min_aoe": 2,
     "max_aoe": 2,
     "flags": [ "SOMATIC", "VERBAL", "CONCENTRATE" ],
-    "min_duration": 30000,
-    "max_duration": 150000,
-    "duration_increment": 4000,
+    "min_duration": 36000,
```

Comment:
I'm guilty of not doing this but I think we should start adding comments to a spell that states the min and max duration in hrs minutes etc instead of only in seconds.   That would make it more human readable, I'm saying.

Reply:
id prefer to have some readable values in the code instead, so not `36000`, but `6 m`

Reply:
I'll make a feature request.